package.json

```diff
@@ -17,6 +17,7 @@
     "babel-jest": "^25.1.0",
     "babel-loader": "^8.1.0",
     "eslint": "^6.8.0",
+    "eslint-plugin-prettier": "^3.1.2",
```

Comment:
Not sure why it is necessary, it should already be installed, but this fixes the failing lint job. Do you have a better fix in mind @mrtnzlml ?

Reply:
Well that's not good, is it? This breaks our eslint config since everyone would have to do it now. 🤔 Why is it merged?

Reply:
I approved it, forgot about the automerging 🙈 we can revert the commit, but I don't know how else to solve it

Reply:
#544